src/inmanta/export.py

```diff
@@ -78,7 +78,7 @@ def __str__(self) -> str:
         return "Cycle in dependencies: %s" % self.cycle
 
 
-def upload_code(conn: protocol.SyncClient, tid: uuid.UUID, version: int, code_manager: loader.CodeManager) -> None:
+def upload_code(conn: protocol.SyncClient, code_manager: loader.CodeManager) -> None:
```

Comment:
This will break the devtools mock, we have to see what the impact will be an how we can make it least painful